modules/fold.cpp

```diff
@@ -0,0 +1,32 @@
+#include "fold.h"
+#include <math.h>
+
+using namespace daisysp;
+
+
+
+void Fold::Init()
+{
+    incr_ = 1000.f;
+    sample_index_ = 0;
+    index_ = 0.0f;
+    value_ = 0.0f;
+}
+
+float Fold::Process(float in)
+{
+    float out;;
```

Comment:
Double semicolon -- surprised this is valid, but everything still builds.

Reply:
It's an advanced C++ technique.